src/app/loadout/ModPicker.tsx

```diff
@@ -117,28 +120,30 @@ function mapStateToProps() {
           (socket) => socket.socketDefinition.reusablePlugSetHash
         );
 
+        // For each of the socket types on the item, figure out what plugs could go into it
+        // and the maximum number of those sockets that can appear on a single item.
         for (const [hashAsString, sockets] of Object.entries(socketsGroupedByPlugSetHash)) {
           const plugSetHash = parseInt(hashAsString, 10);
-          const plugsWithDuplicates: PluggableInventoryItemDefinition[] = [];
-
-          const dimPlugs = filterDimPlugsAvailableToCharacerOrProfile(
+          const dimPlugs = filterDimPlugsUnlockedOnCharacterOrProfile(
             profileResponse,
-            sockets.find((socket) => socket.plugSet)!.plugSet!,
+            sockets[0].plugSet!,
```

Comment:
I wasn't clear what this expression was trying to do

Reply:
Just a partial safety to find a socket with plugset being defined. What you changed it to should be fine.